lib/utils.pm

```diff
@@ -226,21 +226,20 @@ Die, if this is not the case.
 
 =cut
 sub integration_services_check_ip {
-    # Workaround for poo#44771 "Can't call method "exec" on an undefined value"
-    select_console('svirt');
-    select_console('sut');
+    my ($keep_open) = shift // 1;
+
     # Host-side of Integration Services
     my $vmname = console('svirt')->name;
     my $ips_host_pov;
     if (check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
-        $ips_host_pov = console('svirt')->get_cmd_output(
-            'powershell -Command "Get-VM ' . $vmname . ' | Get-VMNetworkAdapter | Format-Table -HideTableHeaders IPAddresses"');
+        (undef, $ips_host_pov) = console('svirt')->run_cmd(
+            'powershell -Command "Get-VM ' . $vmname . ' | Get-VMNetworkAdapter | Format-Table -HideTableHeaders IPAddresses"', keep_open => 0, wantarray => 1);
```

Comment:
```suggestion
            'powershell -Command "Get-VM ' . $vmname . ' | Get-VMNetworkAdapter | Format-Table -HideTableHeaders IPAddresses"', keep_open => $keep_open, wantarray => 1);
```
As long as you have this option, we should use it.

Reply:
this is how it was. I was following the suggestion [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12332#discussion_r616429617). I will change it